test/fixtures/browser-app-2/app.js

```diff
@@ -2,8 +2,10 @@ var loopback = require('loopback');
 var boot = require('../../../');
 
 var app = module.exports = loopback();
+app.start = function(done) {
+  boot(app, {
```

Comment:
Interesting, why is it necessary to move `boot` to `app.start`? Are you expecting all loopback applications to start using this pattern?